File: Override/1.py
class C1:
    def m(self):
        return 'parent'

class C2(C1):
    def m(self):
        result1 = 'child'
        result2 = super().m()
        return result2 + '-' + result1
    pass

o = C2()
print(o.m())

File: Override/1.rb
class C1
  def m()
    return 'parent'
  end
end

class C2 < C1
  def m()
    return super()+'-child'
  end
end

o = C2.new()
p o.m()

# super(): super()가 소속된 메소드의 이름과 같은 부모의 메소드를 가리킨다.
